bot/models/player.py

```diff
@@ -127,11 +138,18 @@ async def show_progress(self, ctx):
 
         letter_xp_string = StringUtil.format_player_xp(self.get_letter_xp())
 
+        try:
+            lvl = self.get_level()
+        except Exception as e:
+            logging.error("# Error 5 #: Error when getting player level")
+            logging.exception(str(e))
+            lvl = 1
+
         embed = discord.Embed(
             title=f"Player profile: {self.get_username()}",
             description=f"""
                 ...................................................
-                Level: **{self.get_level()}**
+                Level: **{lvl}**
```

Comment:
...so that this can instead be `self.get_level()` ...IMHO it's cleaner that way.

Reply:
Done